.circleci/config.yml

```diff
@@ -213,7 +213,7 @@ commands:
     steps:
     # setup
     - amb-linux-install
-    - amb-attach-workspace
+    - amb-checkout
```

Comment:
so i've made the executive decision to not used the cached docker images for the python tests. This makes the build ~5-10 minutes faster and i think it will also make @anodelman 's work to move to github actions a touch easier.....